src/components/vacation-requests-table/vacation-requests-table-toolbar/vacation-requests-table-toolbar.tsx

```diff
@@ -132,7 +133,19 @@ const TableToolbar = ({
                 setValue={setFormOpen}
               />
             </ToolbarGridItem>
-          )}
+          }
+          {selectedRowIds?.length === 1 && 
+          rows.find((request: VacationsDataGridRow) => request.id === selectedRowIds[0])?.status === VacationRequestStatuses.APPROVED &&
+            <ToolbarGridItem sm={adminMode ? 3 : 6} item xs={6}>
+              <FormToggleButton
+                title={strings.tableToolbar.edit}
+                ButtonIcon={Edit}
+                value={formOpen}
+                setValue={setFormOpen}               
+                disabled={adminMode ? false : true}               
```

Comment:
I don't think we wanted anyone (including admins) to be able to edit the vacation request after status change, so why isn't this just true?